crates/bevy_input/src/gamepad.rs

```diff
@@ -1240,6 +1241,104 @@ const ALL_AXIS_TYPES: [GamepadAxisType; 6] = [
     GamepadAxisType::RightZ,
 ];
 
+/// The intensity at which a gamepad's force-feedback motors may rumble.
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct GamepadRumbleIntensity {
+    /// The rumble intensity of the strong gamepad motor
+    ///
+    /// Ranges from 0.0 to 1.0
+    ///
+    /// By convention, this is usually a low-frequency motor on the left-hand
+    /// side of the gamepad, though it may vary across platforms and hardware.
+    pub strong: f32,
```

Comment:
I think that these fields will be clearer out of context if renamed as `strong_motor` and `weak_motor`. Setting a low value for `strong` for example becomes quite confusing otherwise!